### DISCUSSION TITLE
More or unlimited custom music tracks

### DISCUSSION THREAD

**Discussion by TheAtomTwister** [OpenRCT2/OpenRCT2#20203] [Feature requests] [answered]
It'd be cool to have this ability like in RCT3, adding in a lot more custom soundtracks for rides.

**Comment by duncanspumpkin** [comment, marked as answer]
This is the first I've heard of anyone needing more than the 255 limit. How are you using music where you need more than that.

**Reply by ocalhoun6** [reply]
What? Do you *not* use OpenRCT2 to mange your media library?

**Reply by TheAtomTwister** [reply]
Sounds like I've been gone too long.  My bad.  Ignore this post boys, good show. 👍
